paymentsheet/src/test/java/com/stripe/android/paymentsheet/verticalmode/FakeManageScreenInteractor.kt

```diff
@@ -14,10 +14,15 @@ internal class FakeManageScreenInteractor(
                 emptyList(),
                 currentSelection = null,
                 isEditing = false,
+                canDelete = true,
             )
         )
 
     override fun handleViewAction(viewAction: ManageScreenInteractor.ViewAction) {
         viewActionRecorder?.record(viewAction)
     }
+
+    override fun close() {
+        /* Do nothing. */
```

Comment:
I added this comment because our lint check doesn't want there to be an empty code block. Please lmk if there's another solution to this!